pyairwatch/system/users.py

```diff
@@ -18,15 +21,9 @@ def search(self, **kwargs):
             organizationgroupid={locationgroupid}
             role={role}
         """
-        response = self._get(path='/users/search', params=kwargs)
-        return response
-
-    def _get(self, module='system', path=None, version=None, params=None, header=None):
-        """GET requests for the /System/Users module."""
-        response = self.client.get(module=module, path=path, version=version, params=params, header=header)
-        return response
+        return System._get(self, path='/users/search', params=kwargs)
 
-    def _post(self, module='system', path=None, version=None, params=None, data=None, json=None, header=None):
-        """POST requests for the /System/Users module."""
-        response = self.client.post(module=module, path=path, version=version, params=params, data=data, json=json, header=header)
-        return response
+    def create_device_regestration_to_user(self, user_id, register_device_details):
```

Comment:
typo in "regestration"
```suggestion
    def create_device_registration_to_user(self, user_id, register_device_details):
```